05_deploy_bootstrap_vm.sh

```diff
@@ -143,4 +143,27 @@ ssh -o "StrictHostKeyChecking=no" core@$IP sudo podman run \
 # Create a master_nodes.json file
 jq '.nodes[0:3] | {nodes: .}' "${WORKING_DIR}/ironic_nodes.json" | tee ocp/master_nodes.json
 
+MASTER_INTERFACE="eth1"
+
+# Fix etcd discovery on bootstrap
+rm -rf ocp/machineconfigs
+mkdir -p ocp/machineconfigs/temp
+# Find master machine config name
+while [ -z $(ssh -o StrictHostKeyChecking=no "core@$IP" sudo ls /etc/mcs/bootstrap/machine-configs/master*) ]; do sleep 5; done
+
+MASTER_CONFIG=$(ssh -o StrictHostKeyChecking=no "core@$IP" sudo ls /etc/mcs/bootstrap/machine-configs/master*)
+ssh -o "StrictHostKeyChecking=no" "core@$IP" sudo cat "${MASTER_CONFIG}" > ocp/machineconfigs/temp/master.yaml
+# Extract etcd-member.yaml part
+yq -r ".spec.config.storage.files[] | select(.path==\"/etc/kubernetes/manifests/etcd-member.yaml\") | .contents.source" ocp/machineconfigs/temp/master.yaml | sed 's;data:,;;' > ocp/machineconfigs/temp/etcd-member.urlencode
+# URL decode
+cat ocp/machineconfigs/temp/etcd-member.urlencode | urldecode > ocp/machineconfigs/etcd-member.yaml
+# Add a new param to args in discovery container
+sed -i "s;- \"run\";- \"run\"\\n    - \"--if-name=${MASTER_INTERFACE}\";g" ocp/machineconfigs/etcd-member.yaml
+# URL encode yaml
+cat ocp/machineconfigs/etcd-member.yaml | jq -sRr @uri > ocp/machineconfigs/temp/etcd-member.urlencode_updated
+# Replace etcd-member contents in the yaml
+sed "s;$(cat ocp/machineconfigs/temp/etcd-member.urlencode);$(cat ocp/machineconfigs/temp/etcd-member.urlencode_updated);g" ocp/machineconfigs/temp/master.yaml > ocp/machineconfigs/master.yaml
```

Comment:
would it make sense to ensure that the desired substitution has taken place?

Reply:
`etcd-member.urlencode` is fetched by `yq` on L157, so it exists there. A better version would be using `yq` to replace the value, but the cmd line for that is terribly large